### DISCUSSION TITLE
Text editing window size

### DISCUSSION THREAD

**Discussion by tinkererNat** [MicroPad/Web#374] [Ideas]
Just wondering if you've thought about making the text editing window size be able to be set separate from the display size.  
I use a lot of LaTeX equations which in non rendered form take up a lot of line length so it would be nice to make my editing 
window either remember the size it was last resized to using the bottom corner or be able to be have the height and width set
manually (and then remembered when re-editing)

**Comment by NickGeek** [comment]
That's a reasonable idea
